metaflow/plugins/kfp/kfp_foreach_splits.py

```diff
@@ -13,7 +13,7 @@
 )
 
 
-def graph_to_task_ids(graph: FlowGraph) -> Dict[str, int]:
+def graph_to_task_ids(graph: FlowGraph) -> Dict[str, str]:
```

Comment:
We'd have update type info of variables assigned by calls to this function.

Reply:
done